tests/robustness/multiclient_test/multiclient_test.go

```diff
@@ -190,6 +190,19 @@ func TestRandomizedSmall(t *testing.T) {
 	th.RunN(ctx, t, numClients, f)
 }
 
+func TestMaintenanceAction(t *testing.T) {
+	ctx := testlogging.ContextWithLevel(t, testlogging.LevelInfo)
+
+	numClients := 1
+	// Run gc action once for the repository under test.
+	runGCFcn := func(ctx context.Context, t *testing.T) { //nolint:thelper
+		t.Log("run maintenance")
+		_, err := eng.ExecAction(ctx, engine.GCActionKey, nil)
+		require.NoError(t, err)
+	}
+	th.RunN(ctx, t, numClients, runGCFcn)
```

Comment:
Would the following work?

```suggestion
	t.Log("running maintenance directly on the repository under test")

	// bypass the server to directly run maintenance on the repository
	// under test.
	// It launches a kopia process that directly accesses the repository
	// under test using the repo configuration for the server. The
	// server is concurrently running, since the framework starts
	// the server at the beginning of an execution of the framework.
	ctx := testlogging.ContextWithLevel(t, testlogging.LevelInfo)
	_, err := eng.ExecAction(ctx, engine.GCActionKey, nil)

	require.NoError(t, err)
```

Reply:
Verified that the suggestion works.